notebook/notebookapp.py

```diff
@@ -1116,9 +1125,37 @@ def _update_mathjax_config(self, change):
         self.log.info(_("Using MathJax configuration file: %s"), change['new'])
         
     quit_button = Bool(True, config=True,
-        help="""If True, display a button in the dashboard to quit
-        (shutdown the notebook server)."""
+        help=_("""If True, display a button in the dashboard to quit
+        (shutdown the notebook server).""")
+    )
+
+    event_loop_policy = Any(
+        config=True,
+        help=_("""The asyncio event loop policy
+
+        allows specifying an eventloop policy other than the default,
+        e.g.::
+
+            c.NotebookApp.event_loop_policy = 'uvloop.EventLoopPolicy'
+
+        or::
+
+            from tornado.platform.asyncio import AnyThreadEventLoopPolicy
+            c.NotebookApp.event_loop_policy = AnyThreadEventLoopPolicy()
+
+        If specified as a string, the event loop policy class
+        will be imported and instantiated.
+        """
+        ),
     )
+    @observe('event_loop_policy')
+    def _event_loop_policy_changed(self, change):
+        policy = change.new
+        if isinstance(policy, str):
+            # if it's a str, it should be an import string
+            policy = import_item(policy)()
```

Comment:
Doesn't the `Type` traitlet allow importable strings already? Are there cases where it's important to pass an instance so you can instantiate it with non-default arguments?